packages/storybook/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "storybook",
+  "version": "0.0.0",
+  "license": "Apache-2.0",
+  "private": true,
+  "backstage": {
+    "role": "frontend"
```

```suggestion
    "role": "common-library"
```
Maybe this role would suit this package better?

---

It is not a `common-library`. It is not meant to be consumed by other plugins or packages. It is actually an app.

---

According to: https://backstage.io/docs/local-dev/cli-build-system#package-roles:  
>`common-library`: Isomorphic library for use by other packages

This package contains no isomorphic code. It's meant to be used entirely by frontend plugins as a mean to render components individually.